src/psyclone/tests/parse/kernel_test.py

```diff
@@ -52,8 +52,11 @@
 # pylint: disable=invalid-name
 
 # Code fragment for testing standard kernel setup with
-# a type-bound procedure.
-CODE = (
+# a type-bound procedure. This uses the 'iterates_over'
+# member rather than 'operates_on'.
```

Comment:
What is "member"? Why not "metadata element" or just "metadata"?

Reply:
Done.